app/services/media_storage/stored_path.rb

```diff
@@ -11,7 +11,7 @@ module MediaStorage
   # Azure native paths do not need to be rewritten
   module StoredPath
     class << self
-      ENV_PATH_PREFIX = '/' + Rails.env
+      ENV_REMOVAL_REGEX = /\A(?:\/?#{Rails.env}\/?)?(.+)\z/
```

Comment:
1. We dont need to freeze since we have the magic ruby comment at the top
2. Nope, the enforced style according to rubocop config should be slashes

Reply:
>We dont need to freeze since we have the magic ruby comment at the top

magic comment is only for strings...so in theory we could mutate the constant regex (with a warning) without an explicit freeze

> Nope, the enforced style according to rubocop config should be slashes
Interesting, has this changed? it should respect our config, https://github.com/zooniverse/panoptes/blob/4e1add082c779f3f7bc227cae83c37de1a537506/.rubocop.yml#L26-L27

Reply:
The rubocop was changed a little while back, I'm not sure why hound isn't respecting it in regards to the regex.